tests/integration/test_bibtex_exporting.py

```diff
@@ -64,10 +64,14 @@ def test_format_proceeding(app):
     proceeding = get_db_record('lit', 701585)
 
     expected = u'''@proceedings{Alekhin:2005dx,
-      author         = "Alekhin, S. and others",
+      author         = "De Roeck, A. and Jung, H.",
```

Comment:
This also looks weird.

Oh, I understand, the record change on INSPIRE in the meantime: https://inspirehep.net/record/701585.